integration/src/main/java/scripts/testscripts/FolderLifecycle.java

```diff
@@ -111,7 +126,7 @@ protected void doUserJourney(TestUserSpecification testUsers, WorkspaceApi works
     var displayNameLoo = TestUtils.appendRandomNumber("Loo");
     var descriptionLoo = String.format("This is a third-level folder %s", displayNameLoo);
     Folder folderLoo =
-        folderApi.createFolder(
+        folderOwnerApi.createFolder(
```

Comment:
Do you need to cover cases where the folder writer creates a folder? (I think they can) or is that elsewhere?

Reply:
sure. added a case where wrtier create a folder and a case where writer update a folder.